cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ModelUtilsTest.java

```diff
@@ -510,7 +511,7 @@ public void testEmptyTolerations() {
                 .endSpec()
                 .build();
 
-        KafkaCluster model1 = KafkaCluster.fromCrd(kafka, KafkaVersionTestUtils.getKafkaVersionLookup());
+        KafkaCluster model1 = KafkaCluster.fromCrd(new Reconciliation("test", "kind", "namespace", "name"), kafka, KafkaVersionTestUtils.getKafkaVersionLookup());
```

Comment:
Should we have somewhere some static field to use in all of these places? Something like `Reconciliation.DUMMY_RECONCILIATION` which would be statically set to `new Reconciliation("test", "kind", "namespace", "name")`? That way, it would be much easier to for example change the signature later (IDE can probably change it automatically in 200 files, but the review will be again a hell) - with `Reconciliation.DUMMY_RECONCILIATION` you change it just in one place! You should be able to change it in this PR quite easily with search and replace.